localstack/services/sqs/provider.py

```diff
@@ -88,6 +92,13 @@
 LOG = logging.getLogger(__name__)
 
 
+class InvalidAddress(ServiceException):
+    code = "InvalidAddress"
+    message = "The address https://queue.amazonaws.com/ is not valid for this endpoint."
```

Comment:
nit: probably not very important as we don't aim for parity here, but is the address hardcoded? I guess it's the closest exception to what we want to convey, right? In that case I guess it does not really matter.

Reply:
Exactly, just aiming for parity! It's literally what comes out of `aws sqs` when you do anything with an invalid queue URL, regardless of address :shrug: .